docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -604,15 +601,43 @@ To manage the media controller playlist in your application, you must handle req
 3. Register the listener to start receiving notifications about playlist changes:
 
    ```javascript
-   watcherId = mcServerInfo.addPlaylistUpdatedListener(listener);
+   var watcherId = mcServerInfo.playlists.addPlaylistUpdatedListener(listener);
    ```
 
 4. To stop receiving notifications, use the `removePlaylistUpdatedListener()` method:
 
    ```javascript
-   mcServerInfo.removePlaylistUpdatedListener(watcherId);
+   mcServerInfo.playlists.removePlaylistUpdatedListener(watcherId);
    ```
 
+5. To get information about playlists:
+    1. use the `getAllPlaylists()` method:
+
+        ```javascript
+        function successCallback(playlists) {
+            playlists.forEach(function(playlist) {
+                console.log("Playlist name: " + playlist.name);
+            });
+        }
+        function errorCallback(error) {
+            console.log("getAllPlaylists failed with error: " + error);
+            }
+        mcServerInfo.playlists.getAllPlaylists(successCallback, errorCallback);
+        ```
+
+    2. or get playlist by name with method `getPlaylist()`:
+
+        ```javascript
+        /* assuming that the playlist was previously created */
```

Comment:
also here, the existing of playlist with given name is not necessary, code handles error.